cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -3736,20 +3741,20 @@ makeShelleyTransactionBody era@ShelleyBasedEraMary
                              txMintValue
                            } = do
 
-    validateTxBodyContent era txbodycontent
```

Comment:
So you have introduced a lot of unnecessary diffs by changing the indent which makes this PR harder to review because there are now semantic and non-semantic changes. Please undo the indent changes. I know the formatting in cardano-api does not adhere to the style document but if we want to change that, it's best to do that in a separate PR.

Reply:
Ok, indentation restored.

Reply:
Ah I never knew that